### ISSUE TITLE
Querrying Codingjobs does not work (autnes.amcat.nl)

### ISSUE THREAD

**Issue by sebgaly** [amcat/amcat#713]
I'm running into an error when trying to querry codingjobs. I'm working under amcat version 3.5.rc1 ( autnes.amcat.nl )

I select some codingjobs for querrying via the Query section and then click Use in Query. For some codingjobs that works for others I get the following error on the next page and querrying is not possible (i.e. I cannot click the green querry button, Ctrl+Q does nothing):

> An error occured while loading Javascript modules, and this page might not work as expected. This could be due to an adblocker inadvertedly blocking one or more javascript files. Try refreshing the page, and make sure adblockers are disabled for this site.

For these codingjobs for example the error does occur:
http://autnes.amcat.nl/projects/3/query/?jobs=2228,2220,2218,2216,2215,2214,2213,2212

For these it does not and querrying works:
http://autnes.amcat.nl/projects/3/query/?jobs=2217,2208,2207,2198,2197,2195,2194

It definitely is not an Adblocker problem as the error message suggests as there is none installed. Tried refreshing, different browser, different computer, on multiple days, same result.

![screenshot 2018-11-13 13 29 20](https://user-images.githubusercontent.com/33219861/48413518-440a7e00-e748-11e8-9759-ec507915c5de.png)

**Comment by sebgaly** [comment]
Also Ctrl+F5 doesn't work, still the same error. I attached a screenshot of the activity in the network viewer

**Comment by vanatteveldt** [comment]
These are all 200 (=OK), are there any that are not status 200?

**Comment by sebgaly** [comment]
Nope, all 200, but still the error shows and I cannot click query...

**Comment by vanatteveldt** [comment]
@CJStuart  any idea?

**Comment by CJStuart** [comment]
I'm getting the same error. The culprit is a huge number of codes in the schema, causing the JS loader to time out before the select boxes for the code filters are done rendering. I applied a quickfix by delaying the rendering of the codes by 1 second, but a better solution would be to load the codes lazily only after the filter field has been selected. Then they can also be filtered appropriately for that field, so only the relevant codes are shown.

**Comment by sebgaly** [comment]
Thanks for the quickfix! 👍
